k8s.io/test.py

```diff
@@ -120,6 +120,19 @@ def test_code(self):
                 'https://github.com/kubernetes/kubernetes/tree/master/$path',
                 path=path)
 
+    def test_dl(self):
+        # FIXME: https://dl.kubernetes.io is not on the cert
```

Comment:
dl.k8s.io isn't on the cert. I guess we'd need to get a new cert with that added somehow? not sure how to do that.

I just cargo-culted the comment from other test cases. things like https://get.k8s.io are also broken.